exasol_script_languages_container_ci/lib/__init__.py

```diff
@@ -45,3 +47,20 @@ def print_file(filename: Path, writer: Callable[[str], None]):
     """
     with open(filename, "r") as f:
         writer(f.read())
+
+
+def get_files_of_last_commit() -> List[str]:
+    """
+    Returns the files of the last commit of the repo in the cwd.
+    """
+    repo = Repo()
+    commit = repo.head.commit
+    return list(commit.stats.files.keys())
```

Comment:
Generally it is better to return iterators and generators, and let the consumer/user of the function decide if a simple iteration does the job or if he/she needs a materialization and therefore e.g. call the list constructor. 

**TL;DR:** Generators/Iterators are usually way more memory efficient and save memory allocation etc. and threfore often even result in overall better performance.

Reply:
Done